lib/rbac/filterer.rb

```diff
@@ -264,14 +264,17 @@ def search(options = {})
       scope = scope_targets(klass, scope, user_filters, user, miq_group)
               .where(conditions).where(sub_filter).where(where_clause).where(exp_sql).where(ids_clause)
               .includes(include_for_find).includes(exp_includes)
-              .order(order)
+              .order(order || [])
```

Comment:
This line scares me as this is not behind a `inline_view?` unlike the rest.

This might just "be a bug in rbac", but it is potentially changing functionality for EVERY Rbac call, and that leaves me with a bit of concern.

Reply:
Since it seems like this is to solve for the `.select_from_order_columns` loop, you are doing both this change here, and a `.compact` on the passed in array.

Seems like the `.compact` is sufficient for your purposes, so can we leave out this change here?  (if so, update the commit message as well)